src/Npgsql/NpgsqlTypes/NpgsqlTsVector.cs

```diff
@@ -373,7 +374,7 @@ internal set
             public override string ToString()
             {
                 var str = '\'' + (Text ?? "").Replace(@"\", @"\\").Replace("'", "''") + '\'';
-                if (Count > 0)
+                if (_wordEntryPositions != null && _wordEntryPositions.Count > 0)
```

Comment:
Any reason for this? Count isn't good?

Reply:
Will try to workaround it with preconditions.